api_tests/src/utils.ts

```diff
@@ -47,16 +47,19 @@ export async function sleep(time: number) {
 }
 
 export async function timeout<T>(ms: number, promise: Promise<T>): Promise<T> {
```

Comment:
Should we replace our timeout with this?

https://www.npmjs.com/package/p-timeout

Just checked and basically does the same thing as we do here and closes the timeout properly.

Reply:
Do you reckon we should do the same for the SDK then? I'm all for not having to maintain a timeout function haha.

Reply:
> Do you reckon we should do the same for the SDK then?

We can leave that for later I guess, to not introduce more complexity into these changes, but I'm happy to do it in `comit-rs` now.

Reply:
> Do you reckon we should do the same for the SDK then? I'm all for not having to maintain a timeout function haha.

Exactly this, the code duplication makes it obvious that we shouldn't be maintaining this I think.